### PR TITLE
Make some incremental fixes to the aws script

### PR DESCRIPTION
**Pull request by atiannicelli**
### Description:

The following changes have been made to the aws script:

1. Bump version to 0.3.0
2. Remove critical log when finishing
3. Switch output parameter to input for challenge command
4. Make jar and config required parameters
5. Remove s3fs mount logic.
6. Add log to indicate the command to run if disconnected while executing.
7. Add "info" parameter to allow to inject info into an "INFO" file in the output directory.
8. Made input, output, config, and jar parameters require the s3:// preface
9. Remove all single "-" parameters. 

### Potential Impact:

Will require some small changes to the call to execute aws atlas checks.

### Unit Test Approach:

NA

### Test Results:

Executed a few times with different parameter combinations.

### CODE REVIEW

File: scripts/cloud-check-control/cloudAtlasCheckControl.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def __init__(
         mrkey="",
         mrProject="",
         mrURL="https://maproulette.org:443",
-        jar="/home/ubuntu/atlas-checks/build/libs/atlas-checks-*-SNAPSHOT-shadow.jar",
+        jar="atlas-checks/build/libs/atlas-checks-*-SNAPSHOT-shadow.jar",
```
Comment (seancoulter) [inline scripts/cloud-check-control/cloudAtlasCheckControl.py:72]: 🙏

File: scripts/cloud-check-control/cloudAtlasCheckControl.py [hunk 2]
```diff
@@ -601,48 +596,43 @@ def parse_args(cloudctl):
...
         "--mount",
         default=False,
         help="Flag to indicate if s3fs should be used to mount input directory. (Default: False)",
     )
     parser_check.add_argument(
-        "-i",
         "--input",
         required=True,
-        help="IN - The S3 Input directory that contains atlas file directories and sharing.txt. ",
+        help="IN - The S3 Input directory that contains atlas file directories and sharing.txt. "
```
Comment (seancoulter) [inline scripts/cloud-check-control/cloudAtlasCheckControl.py:610]: nit: sharding.txt

File: scripts/cloud-check-control/cloudAtlasCheckControl.py [hunk 3]
```diff
@@ -102,14 +103,34 @@ def __init__(
 
         self.client = None
         self.instanceName = "AtlasChecks"
+        self.localJar = '/tmp/atlas-checks.jar'
```
Comment (seancoulter) [inline scripts/cloud-check-control/cloudAtlasCheckControl.py:106]: Should this be set to the `jar` param?

**Review by seancoulter — COMMENTED** [review]
Nice updates, just noticed a couple minor things

File: scripts/cloud-check-control/cloudAtlasCheckControl.py (hunk 3 above)
Comment (atiannicelli) [reply scripts/cloud-check-control/cloudAtlasCheckControl.py:106]: localJar is where we copy the jar file to if the user gives us a local file or s3 file. It is copied from the remote place to this location for the execution. See "setup_jar" routine.

**Review by seancoulter — APPROVED** [review]

**Review by Bentleysb — APPROVED** [review]

**Pull request merged**